src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -164,7 +163,7 @@ public function jsDependencies(): array
      */
     public function getGateways(): array
     {
-        $enabledGateways = array_keys(give_get_option('gateways'));
+        $enabledGateways = array_keys(give_get_option('gateways_v3'));
```

Comment:
@pauloiankoski I see this we're getting the array keys from this option in a few places here.  We should add a fallback of an empty array or else it's possible to get into a type error from `array_keys(false)`  if the option returns false.  After a migration the chances of this would be slimmer but it's still good practice since we're relying on the database and and using a php function that needs an array.

```
array_keys(give_get_option('gateways_v3', []))
```